CMakePresets.json

```diff
@@ -270,7 +273,13 @@
       "name": "ci-gcc-release",
       "inherits": [
         "gcc-release"
-      ]
+      ],
+      "cacheVariables": {
+        "CMAKE_INTERPROCEDURAL_OPTIMIZATION": "TRUE"
+      },
+      "environment": {
+        "ASAN_OPTIONS": "detect_odr_violation=0"
```

Comment:
do we want to re-enable this since we found the `static constexpr lambda` thing? or is it a completely separate issue?

Reply:
This is a separate issue.